lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -438,8 +438,11 @@ sub _show {
 sub job_next_previous_ajax ($self) {
     return $self->reply->not_found unless my $main_job = $self->_get_current_job;
     my $main_jobid = $main_job->id;
-    my $p_limit = $self->param('previous_limit') // 400;
-    my $n_limit = $self->param('next_limit') // 100;
```

Comment:
I suggest to keep this original values as default so instead of 250/250 use 400/100 in the config file, Setup.pm, etc.

Reply:
Yes, unless you did some performance testing the would show 250/250 is in fact a better choice I'd also keep the current defaults.